### PR TITLE
RTDT-1844_object_deteciton_model_universal_batch

### PR DESCRIPTION
**Pull request by Andrii-Sheba**
# old approach:
Screenshot from 2024-08-05 18-05-40

# new, dynamic_export:
Screenshot from 2024-08-05 18-05-57

# Other current models are also dynamic

- kp model
image

- cls model
image

### CODE REVIEW

File: auto_training/export.py [hunk 1]
```diff
@@ -129,11 +129,11 @@ def parse_args():
...
     if args.shape is None:
         img_scale = cfg.test_pipeline[1]['img_scale']
-        input_shape = (1, 3, img_scale[1], img_scale[0])
+        input_shape = (4, 3, img_scale[1], img_scale[0])
     elif len(args.shape) == 1:
-        input_shape = (1, 3, args.shape[0], args.shape[0])
+        input_shape = (4, 3, args.shape[0], args.shape[0])
     elif len(args.shape) == 2:
-        input_shape = (1, 3) + tuple(args.shape)
+        input_shape = (4, 3) + tuple(args.shape)
```
Comment (jplapp) [inline auto_training/export.py]: are these changes needed if we use dynamic export anyway?

**Review by jplapp — APPROVED** [review]

File: auto_training/export.py (hunk 1 above)
Comment (Andrii-Sheba) [reply auto_training/export.py]: thanks, we don`t use this param at dynamic mode.
